doc/basicopt.txt

```diff
@@ -31,6 +31,9 @@ Options:
   --app:console|gui|lib|staticlib
                             generate a console app|GUI app|DLL|static library
   -r, --run                 run the compiled program with given arguments
+  -e, --eval:cmd            evaluates nim code directly; e.g.: `nim -e:"echo 1"`
```

Comment:
Please only introduce the long version, I cannot remember all these single char abbreviations anymore.

Reply:
done